pkgs/libresoc/libresoc-pyelftools.nix

```diff
@@ -0,0 +1,13 @@
+{
+  python39Packages,
+  fetchgit,
+}:
+python39Packages.pyelftools.overrideAttrs (_: rec {
+  name = "libresoc-pyelftools";
+  version = "v0.30";
+  src = fetchgit {
+    url = "https://git.libre-soc.org/git/pyelftools.git";
```

Comment:
use libre-soc's fork of pyelftools because a few commits have diverged from pyelftools packaged by nixpkgs